server/channels/app/notification_push.go

```diff
@@ -490,6 +490,10 @@ func (a *App) rawSendToPushProxy(msg *model.PushNotification) (model.PushRespons
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != http.StatusOK {
```

Comment:
@larkox Are we okay with just checking that its not a 200? I think that's the only code we expect it to send.

Reply:
Should we count this as an error in the metrics ? Or are we already doing that in the top level caller ?

Reply:
Re: only checking 200

Looking at the code, we always return a 200, so this will solve any "network layer" issue (which is what we suppose we have here).

If there is anything at the "app layer" would be that for whatever reason we couldn't write the message into the response (which should never happen, and will be logged in the logs as "Failed to write response").

Reply:
@marianunez It's happening here: https://github.com/mattermost/mattermost/blob/master/server/channels/app/notification_push.go#L188